### ISSUE TITLE
Compilation error - missing CaptchaValidatorSvc

### ISSUE THREAD

**Issue by KrzysztofMadejski** [petabyte-research/redflags#14]
```
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /home/redflags/redflags/redflags-web/src/main/java/hu/petabyte/redflags/                                                                          web/svc/SecuritySvc.java:[39,28] cannot find symbol
  symbol:   class CaptchaValidatorSvc
  location: class hu.petabyte.redflags.web.svc.SecuritySvc
```

**Comment by dzierzy** [comment]
missing file added

**Comment by KrzysztofMadejski** [comment]
Closed in 6e49802a6c85b538f007378dcc037051055656a8
